src/ess/estia/conversions.py

```diff
@@ -59,12 +62,21 @@ def divergence_angle(
 
 
 def wavelength(
```

Comment:
Hmm, shouldn't there already be a function to compute wavelength in the graphs that scippneutron provides? Or do we still need to repeat it here?